aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/helpers/SignInChallengeHelper.kt

```diff
@@ -78,7 +78,7 @@ internal object SignInChallengeHelper {
             }
             challengeNameType is ChallengeNameType.SmsMfa ||
                 challengeNameType is ChallengeNameType.CustomChallenge ||
-                challengeNameType is ChallengeNameType.NewPasswordRequired -> {
+                challengeNameType is ChallengeNameType.NewPasswordRequired || challengeNameType?.value == "RESET_PASSWORD" -> {
```

Comment:
I am not sure if something like this exists. Cognito doesn't have this challenge as a type. `challengeNameType?.value == "RESET_PASSWORD" `

Reply:
It does not, I am sending it when the exception is raised.